### DISCUSSION TITLE
how to get all available functions and their paramaters in the libproducts?

### DISCUSSION THREAD

**Discussion by SeqCrafter** [JuliaPackaging/BinaryBuilder.jl#1265] [Q&A]
I want to know if there is a simpler way than looking at the header file in the source code to get all the functions/api and their usage that are defined in libproducts. So I can use ccall to use it

**Comment by giordano** [comment]
BinaryBuilder doesn't deal with bindings generation, that's a separate business. But you can use other tools like [`Clang.jl`](https://github.com/JuliaInterop/Clang.jl) to automate the process.
